src/commands/chains.ts

```diff
@@ -14,5 +14,5 @@ export const builder: CommandBuilder<{}, Options> = (yargs) =>
 const chains = getChains();
 
 export const handler = async (_argv: Arguments<Options>): Promise<void> => {
-  console.dir(chains, { depth: null });
+  console.log(JSON.stringify(chains, null, "\t"));
```

Comment:
No need to format here, just leave it "raw" and we can let downstream consumers prettier it as required.